src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.Generated.cs

```diff
@@ -250,12 +204,6 @@ public ActionCallInstruction(MethodInfo target)
             _target = (Action)target.CreateDelegate(typeof(Action), target);
         }
 
-        public override object Invoke()
```

Comment:
Note that all logic for these is in the `Run` method; the `Invoke` method is never called . Also, this stuff is conditionally compiled using `FEATURE_FAST_CREATE` which is not used at all today.